src/multiselectMixin.js

```diff
@@ -318,9 +320,6 @@ module.exports = {
       this.adjustSearch()
     },
     'search' () {
-      /* istanbul ignore else */
-      if (this.search === this.currentOptionLabel) return
```

Comment:
I had to delete this line so that when you delete the search query, a search event will be emitted with an empty string value.